src/sentry/eventstream/base.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+
+from sentry.utils.services import Service
+
+from sentry.tasks.post_process import post_process_group
+
+
+class EventStream(Service):
+    __all__ = (
+        'publish', 'consume',
+    )
+
+    def publish(self, event, primary_hash, **kwargs):
```

Comment:
These signatures are effectively temporary until @tkaemming rolls in and changes things up.

Reply:
So, this actually won't work with the delegator right now due to the `**kwargs`: https://github.com/getsentry/sentry/blob/53837d9e8dad2443c3881fcd05f1b261bdd922fd/src/sentry/utils/services.py#L285-L293

It's because this doesn't hold:

```python
>>> import inspect
>>> f = lambda x, y, *a, **k: (x, y, a, k)
>>> f(0, 1, 2, a=3, b=4) == inspect.getcallargs(f, 0, 1, 2, a=3, b=4)
False
```

I'm going to need to fix that regardless but if you want to get this out before I have a solution, it'd be better to just make all of these arguments explicit in the function definition. I think what you have now is fine in `src/sentry/event_manager.py`.